pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -181,6 +188,14 @@ func (p Provisioner) Grant(options *apibkt.BucketOptions) (*bktv1alpha1.ObjectBu
 		p.deleteOBCResourceLogError("")
 		return nil, err
 	}
+
+	// setting quota limit if it is enabled
+	err = p.setAdditionalSettings(options)
+	if err != nil {
+		p.deleteOBCResource("")
```

Comment:
same here.

Reply:
@subhamkrai : will send separate PR to address the issue in master branch. Thanks for pointing out

Reply:
@subhamkrai : https://github.com/rook/rook/pull/5935